openfisca_aotearoa/variables/acts/social_security/sole_parent_support/sole_parent_support.py

```diff
@@ -73,7 +73,7 @@ class sole_parent_support__meets_relationship_qualification(variables.Variable):
     """
     def formula(persons, period, parameters):
         # Do they have a partner
-        no_partners = (persons("person_has_partner", period) == 0)
+        no_partners = (persons("person_has_partner", period.first_week) == 0)
```

Comment:
is this accurate with the legislation? Could we add a comment here for future devs to understand why we only check the first week?